modules/ovn-kubernetes-architecture-con.adoc

```diff
@@ -12,16 +12,16 @@ The key components are:
 * **Cloud Management System (CMS)** - A platform specific client for OVN that provides a CMS specific plugin for OVN integration. The plugin translates the cloud management system's concept of the logical network configuration, stored in the CMS configuration database in a  CMS-specific  format, into an intermediate representation understood by OVN.
 * **OVN Northbound database (`nbdb`)** - Stores the logical network configuration passed by the CMS plugin.
 * **OVN Southbound database (`sbdb`)** - Stores the physical and logical network configuration state for OpenVswitch (OVS) system on each node, including tables that bind them.
-* **ovn-northd** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-master` pods.
+* **ovn-northd** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-node` pods.
```

Comment:
* Existing content, but would be good to have a noun after `nbdb` and `sbdb` (not sure what they are aside from "components").
* In another module, you have "OVN north daemon" in plain text. Could this match the other list items above like so? 

```suggestion
* **OVN north daemon (`ovn-northd`)** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-node` pods.
```